app/posts/creating-successful-govuk-tools.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Successfully creating tools to help build GOV.UK services
+description: Thoughts on what it takes to make something other teams will use
+date: 2024-01-16
+author:
+  name: Peter Yates
+  url: https://github.com/peteryates
+image:
+  src: /assets/posts/creating-successful-govuk-tools/illustration.png
+  alt: Red ruby gemstone, with multiple colours bursting out from behind.
+  opengraphImage: true
+---
+
+I joined the [Get School Experience](https://schoolexperience.education.gov.uk/) team in 2019 and it was my first role in a government department. Our biggest challenge was to work out how to build a multi-page form and I was surprised there wasn't already an established pattern. There wasn't even a [Simple Form](https://github.com/heartcombo/simple_form) config.
+
+Despite only having worked in DfE Digital for a few weeks I'd seen just how important collaboration was. I remember how refreshing it was that working collaboratively was the default, and that was reinforced by [point 13 of The GOV.UK Service Manual](https://www.gov.uk/service-manual/service-standard/point-13-use-common-standards-components-patterns).
+
+> If you develop your own patterns or components, share them publicly so others can use them.
+
+I'd found a gap. Something I thought should exist but didn't. Something I was certain others would find useful. Something I thought I could do.
+
+I decided to have a go at building and sharing a form builder and spent my free time over the next month working on it. I announced it [via a very nervous-sounding YouTube screencast](https://www.youtube.com/watch?v=PhoFZ0qXAlA) and, thankfully, it was well received.
+
+It was my first time building and publishing a library before, but having used plenty I had a good idea of what to prioritise if I wanted to make it a success. Here's my list:
+
+### 1. Active development
+
+Unmaintained repositories are likely to become a burden, and might even hold you back from upgrading your application. It's a huge red flag. Even if new features aren't constantly being added, keeping on top of dependencies and making sure the documentation is up to date will give visitors to your project's page some confidence it's being looked after.
+
+### 2. A responsive maintainer
+
+Knowing there's someone who will respond to queries on Slack, fix bugs and keep the library up to date and in sync with the design system is a must.
+
+### 3. An intuitive design
+
+There was already a reference implementation written in [Nunjucks](https://mozilla.github.io/nunjucks/) that would provide inspiration but this had to feel like Rails. We use blocks instead of strings of HTML, we use `snake_case` instead of `camelCase`. If the ergonomics weren't right developers wouldn't use it.
```

Comment:
```suggestion
There was already a reference implementation written in [Nunjucks](https://mozilla.github.io/nunjucks/) that would provide inspiration but this had to feel like Rails. We use blocks instead of strings of HTML, we use `snake_case` instead of `camelCase`. If the ergonomics weren't right, developers wouldn't use it.
```